src/rest/APIRouter.ts

```diff
@@ -0,0 +1,71 @@
+import type { RequestInit } from "node-fetch"
+import type APIClient from "./APIClient"
+
+type Serialize = () => string
+type APIRequest<T> = (options?: RequestInit) => T
+type Proxy = () => void
+
+interface APIHandler<T> {
+    get(target: never, name: string): Serialize | APIRequest<T> | Proxy
+    apply(target: never, _: never, args: string[]): Proxy
+}
+
+/**
+ * @license https://github.com/discordjs/discord.js/blob/master/LICENSE
+ * @description 
+ * I, zero734kr, maintainer and lead developer of js-sdk of KOREANBOTS, declare 
+ * that this part of source code 'APIRouter.ts' is taken from 
+ * https://github.com/discordjs/discord.js/blob/master/src/rest/APIRouter.js,
+ * and modified the original source code to be compatible with TypeScript 
+ * and KOREANBOTS API Routes. There were not many changes with original source code.
+ * @see https://github.com/discordjs/discord.js/blob/master/src/rest/APIRouter.js
+ */
+
+// eslint-disable-next-line @typescript-eslint/no-empty-function
+const noop = () => { }
+const methods = ["get", "post", "delete", "patch", "put"]
+const userIdRegex = /\d{16,19}/g
```

Comment:
This regex is duplicated it would be better to maintain by separating Constant file